### DISCUSSION TITLE
[x][closed] Should lab and compute protocol info fields rather be IRIs?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#87] [Team Posts]
E.g. ENCODE provides URL for pipelines, etc..

**Comment by sveinugu** [comment]
Daniel's answer: That's be great, but I think that would be a high bar to entry.

**Comment by jmfernandez** [comment]
I also agree with @dzerbino

**Comment by sveinugu** [comment]
@dzerbino I added a recommentation about this in the "description" field. We cannot require it, I agree.
